doc/refman/RefMan-tac.tex

```diff
@@ -4710,6 +4750,13 @@ \subsection{\tt congruence}
     described above.
 \end{ErrMsgs}
 
+\noindent {\bf Remark: } {\tt congruence} can be made to print debug
```

Comment:
What's the difference between `\Rem` and `\noindent {\bf Remark: } `, and why choose one over the other?

Reply:
I have no idea, IIRC I just picked whichever I saw first (without caching past picks).

Reply:
Probably irrelevant anyways since we are soon moving to Sphinx.